README.md

```diff
@@ -28,6 +28,18 @@ https://www.microsoft.com/en-us/download/details.aspx?id=30679
 
 &nbsp;
 
+### I added an object and my `fps_real` went from 5,000 down to 4,000. Help!
+
+Don't worry about it. `fps_real` is deceptively useless and the only thing it's really good for is making people panic needlessly.
+
+Frame rate is [inversely proportional to frame time](https://www.desmos.com/calculator/d4hvus9oys), which means that (among other things) minute differences in very small frame times will have seemingly disproportionately large effects on the `fps_real`. What you really care about is the **frame time,** or the amount of time it takes to deliver a frame. This means you have a budget of about 16.6 ms to do all of your Step and Draw event processing if you want to maintain a refresh rate of 60 frames per second. An `fps_real` value of 5,000 means that your entire game is finished with its update tick in **0.2 milliseconds,** and an `fps_real` value of 4,000 means that your entire game is finished with its update tick in **0.25 milliseconds.** That's a difference of **0.05 milliseconds,** which profesional game developers like to call "a vanishingly tiny proportion of your frame time budget" (it's about 0.3%).
+
+Whatever you did to make your `fps_real` dip, you can do it another 300 times and still hit a 60 fps target.
+
+To put it in perspective, it takes a photon of light about 16 milliseconds to travel from New York to San Francisco. In the interval of time between 5,000 fps_real and 4,000, the photon is still in East Orange, New Jersey.
```

Comment:
Maybe mentioning, that we can get how much of frame game eats is
```gml
(1000/fps_real)/(1000/game_get_speed(gamespeed_fps))
```
or sth like that, and as long as it's < 1 it's good ?